### DISCUSSION TITLE
Use an endpoint instead of a port for the Minio service?

### DISCUSSION THREAD

**Discussion by mcg1969** [nebari-dev/nebari#2845] [Ideas]
Hey folks! I wanted to ask if there was a specific reason why the Minio service was chosen to be deployed on a dedicated _port_ (9080) instead of somewhere on the path of the existing HTTPS port (443). In my experience, as long as the bucket name does not conflict with a path used by other applications, the ingress (e.g., Traefik) can handle routing S3 traffic just fine.

Reducing the use of custom ports will make it easier to deploy Nebari in situations where it is important to re-use an existing cluster's ingress.

**Comment by Adam-D-Lewis** [comment]
We discussed this on an internal call today.  The consensus was it we think this should be possible to change.  There may be an issue with conda store supporting minio on a subpath, but we agree it can/should be fixed if not.  Thanks for calling this out and I'll transfer this to an issue.

**Reply by mcg1969** [reply]
Since a standard S3 url always includes the bucket name at minimum on the path I suspect conda-store will be fine.

And if the bucket name does not conflict with other API paths you can put the bucket names directly in the ingress so the minio "base" has no subpath

**Comment by viniciusdc** [comment]
Hi @mcg1969, the discussion was moved to #2848 -- I overlooked the cross-reference not showing up here

**Comment by Adam-D-Lewis** [comment]
I'm locking conversation if favor of moving further conversation to the issue.  Thanks for everyone's comments on this.
